components/common/authProcess.js

```diff
@@ -159,6 +164,11 @@ const AuthProcess = ({
             Success!
           </Heading>
           <Text>{successText}</Text>
+          {confirmationCode && (
+            <Subheading>
+              {`Confirm the following code: ${confirmationCode}`}
```

Comment:
can you check to see how Vercel words it ? (no change necessary for approval of this PR, but i think this copy will be confusing)

Reply:
done